python/py-MDAnalysis/Portfile

```diff
@@ -4,13 +4,13 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-MDAnalysis
-version             1.0.0
+version             2.0.0
 revision            0
 categories-append   science
 platforms           darwin
 license             GPL-2+
 
-python.versions     27 36 37 38
+python.versions     27 36 37 38 39
```

Comment:
Just to point out - we got rid of py2.7 as of 2.0.0.

Reply:
@btywoniuk if you want to retain PY27 you'll have to add some additional code within the ` if {${name} ne ${subport}} { ` block:
```
    if  {${python.version} eq 27} {
        version     ????
        revision    0

        checksums   rmd160  ??? \
                    sha256  ??? \
                    size    ???
    }
```

Reply:
In https://github.com/macports/macports-ports/pull/12970/commits/61130190b0c974a6b9975bd0c6cddacb75f951fa  I fully removed python 2.7 from a list of supported versions for this package.